lib/bridge/github/contents.rb

```diff
@@ -16,13 +16,14 @@ def issue_template
 
     def issue_template_content
       begin
-        template = Base64.decode64(issue_template['content'])
+        template = issue_template
+        content = Base64.decode64(template['content']) if template
       rescue => e
-        ::Raygun.track_exception(e, custom_data: {error_response: template.to_s})
-        template = nil
+        ::Raygun.track_exception(e, custom_data: {error_response: content.to_s})
```

Comment:
This seems like it should stay `template`? And probably guard against `nil`, too.